omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -52,13 +52,21 @@
     # Whitelist the docs necessary to serve up error pages and friendly
     # html to non-chef clients hitting this host.
     location ~ "^/[0-9]{3,3}\.(json|html)|favicon.ico|index.html$" {
+      add_header Content-Security-Policy "default-src 'self';";
```

Comment:
only adding self here would mean if the request was from different sub-domain these options would not get added. Perhaps testing with manage would clarify it.

Reply:
Tested with Manage. Works great still